test/PlayerActions/PlayerActions.test.js

```diff
@@ -0,0 +1,39 @@
+const expect = require("chai").expect;
+const EventEmitter = require("events");
+const sinon = require("sinon");
+
+const mockUIFactory = require('../mocks/UI');
+
+const { GameState } = require("../../entities/GameState");
+const { createPlayer } = require("../testutils");
+
+describe("PlayerActions", function () {
+  let gameState;
+  let mockUI;
+  let eventBusEmitter;
+  let PLAYER_ACTIONS;
+
+  beforeEach(() => {
+    gameState = new GameState();
+    eventBusEmitter = new EventEmitter();
+    mockUI = mockUIFactory();
+
+    gameState.players = [createPlayer({ name: "player1" })];
+    PLAYER_ACTIONS = require("../../entities/PlayerActions")(eventBusEmitter, mockUI, gameState);
+  });
+
+  afterEach(() => {
```

Comment:
Do we use sinon here? could remove potentially

Reply:
Correct - if we move the other tests into modules then load them here, we'll restore this, but this can go away for now